MarkBind/markbind
# Remove scroll-to-top button from print view

**What is the purpose of this pull request?**


This PR provides a quick fix that resolves #1438. 



**Overview of changes:**
Add a new global CSS rule for print media to hide any HTML elements with `no-print` class. 

**Anything you'd like to highlight / discuss:**
Whether the CSS rule should be placed elsewhere. 

**Testing instructions:**
`markbind serve` and ensure that the scroll-to-top button is in view on the browser. Open up the printing menu, you should notice that the scroll-to-top button is not visible in the print preview. It should not appear on the exported PDF file as well. 



**Proposed commit message: (wrap lines at 72 characters)**
Remove scroll-to-top button from print view

It is unlikely that users would want the scroll-to-top button to 
be visible in the printed version of the website. 

Let's enhance the scroll-to-top button feature by removing 
it from the printed version. 



---

**Checklist:** :ballot_box_with_check:
